atomic_reactor/plugins/exit_remove_worker_metadata.py

```diff
@@ -38,4 +38,4 @@ def run(self):
                 osbs.delete_config_map(cm_key)
                 self.log.debug("ConfigMap %s deleted", cm_key)
             except OsbsResponseException as ex:
-                self.log.warning("Failed to delete ConfigMap %s: %s", cm_key, ex.message)
+                self.log.warning("Failed to delete ConfigMap %s: %s", cm_key, str(ex))
```

Comment:
`str()` is not  needed, `%s` in format string will call `str()`

Reply:
Ok, fixed